### DISCUSSION TITLE
Version check/update after security concern

### DISCUSSION THREAD

**Discussion by lightzall** [sumatrapdfreader/sumatrapdf#2999] [General]
I read somewhere Sumatra was being piggybacked and redistributed with backdoor and account passwords were comprommised.  That may not be true or its out-dated so apologies if I am off the mark but when I checked the version I had for update it said 3.3.3 x64 was latest version and I think i may have used a bunk install. I will uninstall win10x64 3.3.3 version and use the 3.4.6x64 now but wondering if there was an issue or any steps to take. Thanks.

**Comment by GitHubRulesOK** [comment]
release update checks can be delayed "years 🙂 " 3.1.2 may still show as latest stable especially for XP users, and 3.3.3 offered for 3.0+ updates i.e. most others, however 3.4.6 should for many users be considered stable (unless your a PDFLaTeX user) current 3.5.15244 is most stable pre-release and 3.5.15249 fails (crashes) on update thus at any time is not easy to say which is best for a given user (Backup older stable copies before updates).

Always download from https://www.sumatrapdfreader.org/download-free-pdf-viewer and test sha256 validates with others at Virustotal only occasionally may raise a false positive for example that 3.4.6x64 installer is clean as https://www.virustotal.com/gui/file/aa79391c7db478fbb969875da39ce09e3e8124b869acc3178f5b6a3b4e10d5ce and look for signing detail thus ignore Verdict: LIKELY_MALICIOUS by file IO their detection sees the traffic caused by updates, downloads AS 20940 ( Akamai International B.V. ) and MS components [msidentity.com](https://www.virustotal.com/gui/domain/msidentity.com) and other such as secure signing !! By their criteria every copy of any devices OS should be removed.

 Posting SHA on own download site is no good if the site is compromised, only confirms the suspect download was not corrupted, so odd others do that !

**Reply by lightzall** [reply]
Thank you for the reponse. I tried another another program as a means to modify, collate or write to existing pdfs. I dont think it was open source and it was not working for me.  Since this is open source but specifically just a reader is there an extension or other way to write to pdfs within the sumatra reader? I am using Libre office to write stuff and converting docx and other word pages into more cohesive multi-page single item. Specifically I am hoping to use pdf versions of court rulings and annotate over existing pages or modify text in source document.  Nothing need to be printed but consolidating more than two or three multipage decisions on the same case together as a single document or folder with good basic copypasta function would be perfect

**Reply by GitHubRulesOK** [reply]
The pre-release   has the latest annotation updates (current one has a bug that needs for now a workaround) but they are highlights and basic comments thus currently limited to no more than 256 characters at best. For larger notes you have Edge but those are additions to PDF not full edits.
 Editors are expensive to produce and much heavier coding than readers. The best suggestion is the fact that SumatraPDF and Tracker PDF x-change work well together as a PDF page can be launched/shuttled between them for fast read and slower edits.

**Reply by rwiesbach** [reply]
> release update checks can be delayed "years slightly_smiling_face " 3.1.2 may still show as latest stable especially for XP users, and 3.3.3 offered for 3.0+ updates i.e. most others, however 3.4.6 should for many users be considered stable (unless your a PDFLaTeX user) 

Today I noticed that current stable is 3.4.6 and I have 3.3.3 (> 1 year old) installed which keeps telling me, that there is no newer version (update check within the app). If 3.4.6 is not stable for PDFLaTeX users, why is 3.4.6 the version offered as stable on the website? On the other hand: If it is offered as current stable on the website, why is it ignored by the internal update check? Have there been no security fixes in 3.4.x that are relevant for 3.3.x as well? Does update check only consider security (not feature) updates? I am confused, please help (and even better: provide more information on the download page and during the internal update check)

Thanks

**Reply by GitHubRulesOK** [reply]
3.4.6 is latest of many stable versions and has a known problem for some (not all latex users) thus I should not have indicated as not stable, let me rephrase:-
 **3.4.6 is stable with known issues for some 64bit synctex users** is offered update for prior versions
3.3.3 is last known stable for many users and may not auto update but be offered as such for prior versions.
3.1.2 is last known stable for XP users (but they may be auto offered 3.3.3 so should avoid it)

If you have a need for running synctex as 64bit use 3.4.6 32bit by all means or alter the editors synctex settings.
